client/src/components/LevelLayout.tsx

```diff
@@ -37,11 +52,14 @@ const LevelLayout: React.FC<Props> = ({
           Previous Step
         </Button>
         {game.stepIndex === game.steps.length - 1 && game.level < 5 - 1 ? (
+            <Flex>
           <Button onClick={() => game.jumpToLevel(game.level + 1)}>
             Next Level
           </Button>
+           <Confetti recycle={false} />
+           </Flex>
         ) : game.stepIndex !== game.steps.length - 1 && (
-          <Button onClick={game.nextStep}>Next Step</Button>
+          <Button onClick={() => {game.nextStep(); didWin();}}>Next Step</Button>
```

Comment:
nit: This looks a bit odd. Consider creating a `handleNextStep` function.

Reply:
Great, I added a helper function